guide/src/wasm-bindgen-test/continuous-integration.md

```diff
@@ -0,0 +1,73 @@
+# Setting Up Continuous Integration with `wasm-bindgen-test`
+
+This page contains example configurations for running `wasm-bindgen-test`-based
+tests in various CI services.
+
+Is your favorite CI service missing? [Send us a pull
+request!](https://github.com/rustwasm/wasm-bindgen)
+
+## Travis CI
+
+```yaml
+INSTALL_NODE_VIA_NVM: &INSTALL_NODE_VIA_NVM
+  |
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+    source ~/.nvm/nvm.sh
+    nvm install v10.5
```

Comment:
I think this isn't necessarily required actually, the version of `node` installed automatically on Travis should already have wasm support

Reply:
Node comes with travis, even for non-node/js projects?

Reply:
I believe so! I could be wrong though, I'd have to check...